.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:latest
+    - image: circleci/node:16.13
```

Comment:
is there here any limitation to keep it latest?

Reply:
Yes, it doesn't work on latest because of some internal changes.  This allows it to compile ont he circleci machines.